frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -34,8 +38,12 @@ const userRoleAssignments = shallowRef<
     FamApplicationUserRoleAssignmentGet[]
 >(props.userRoleAssignments);
 
+const passThroughTabStyles = {
```

Comment:
this variable doesn't seem in use?

Reply:
yeah, it was a left over, Thanks for pointing out